Sources/App/Core/AppVersion.swift

```diff
@@ -1 +1,4 @@
-let appVersion = "dev - will be overriden in release builds"
+// In a local development environment, appVersion will remain nil (as set here).
+// In the staging development environment, appVersion is set to the commit hash of the deployed version.
+// In production, appVersion is set either to released tag name, or to the commit hash if that does not exist.
+let appVersion: String? = nil
```

Comment:
I suspect this will fail the builds on Gitlab, where we create the images, because the command is:

```
  script: |
    VERSION=${CI_COMMIT_TAG:-$CI_COMMIT_SHA}
    echo 'let appVersion = "'${VERSION}'"' > ./Sources/App/Core/AppVersion.swift
```

This would need to be changed to

```
  script: |
    VERSION=${CI_COMMIT_TAG:-$CI_COMMIT_SHA}
    echo 'let appVersion: String? = "'${VERSION}'"' > ./Sources/App/Core/AppVersion.swift
```

Reply:
Fixed